src/bundle/Resources/public/js/scripts/admin.contenttype.selection.js

```diff
@@ -15,12 +15,21 @@
 
             container.querySelector(SELECTOR_BTN_REMOVE)[methodName]('disabled', disabledState);
         };
+        const getNextId = () => {
+            if (countExistingOptions() === 0) {
+                return 0;
+            }
+            const lastInput = container.querySelector(SELECTOR_OPTIONS_LIST).lastElementChild.querySelector('input[type="text"]');
+            const lastId = parseInt(lastInput.name.match(/.+\[(\d)]$/)[1]);
```

Comment:
This should be stored in data-attribute. The same like @Nattfarinn did in ezmatrix.

Reply:
@dew326 
fixd